install/install.go

```diff
@@ -128,23 +191,41 @@ func checkPods(c *k8s.Client, o Options) {
 	fmt.Printf("😋\tChecking if KubeArmor pods are running...\n")
 	stime := time.Now()
 	otime := stime.Add(600 * time.Second)
-	for {
-		time.Sleep(200 * time.Millisecond)
-		pods, _ := c.K8sClientset.CoreV1().Pods("").List(context.TODO(), metav1.ListOptions{LabelSelector: "kubearmor-app", FieldSelector: "status.phase!=Running"})
-		podno := len(pods.Items)
-		fmt.Printf("\r\tKubeArmor pods left to run : %d ... %s", podno, cursor[cursorcount])
-		cursorcount++
-		if cursorcount == len(cursor) {
-			cursorcount = 0
-		}
-		if !otime.After(time.Now()) {
-			fmt.Printf("\r⌚️\tCheck Incomplete due to Time-Out!                     \n")
-			break
+
+	if !o.Legacy {
+		for {
+			time.Sleep(900 * time.Millisecond)
+			pods, _ := c.K8sClientset.CoreV1().Pods("").List(context.TODO(), metav1.ListOptions{LabelSelector: "kubearmor-app=kubearmor", FieldSelector: "status.phase==Running"})
+			podno := len(pods.Items)
+			fmt.Printf("\rℹ️\tWaiting for KubeArmor Daemonset to run:  %s", cursor[cursorcount])
+			cursorcount++
```

Comment:
Let's have independent checkpods function for legacy installation. Let's not couple both the logic in if else.